bin/resource_management/resource-broken-links-csv.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
```

Comment:
?

Reply:
oh did you paste this in from a jupyter notebook?

Reply:
Yes I wrote it in a notebook and exported from there